config/webpack.config.js

```diff
@@ -24,12 +24,26 @@ module.exports = {
                 exclude: /node_modules/,
                 loader: "babel-loader",
                 options: {
-                    presets: [
-                        'env',
-                        'stage-0',
-                        'es2015',
-                        'react'
+                    "presets": [
+                        [
+                            "env",
+                            {
+                                "targets": {
+                                    "browsers": [
+                                        "> 1%",
+                                        "last 2 versions"
+                                    ],
+                                },
+                                "loose": true,
+                                "useBuiltIns": true,
+                            }
+                        ],
+                        "react"
                     ],
+                    "plugins": [
+                        "transform-object-rest-spread",
```

Comment:
since we added `plugins`, we need to add them as deps:

```
yarn add -D babel-plugin-transform-object-rest-spread babel-plugin-transform-class-properties
```